core/Cargo.toml

```diff
@@ -16,7 +16,7 @@ fnv = "1.0"
 futures = { version = "0.3.28", features = ["executor", "thread-pool"] }
 futures-timer = "3"
 instant = "0.1.11"
-libp2p-identity = { version = "0.1", path = "../identity", features = ["peerid", "ed25519"] }
+libp2p-identity = { version = "0.1.3", path = "../identity", features = ["peerid", "ed25519"] }
```

Comment:
Why do we need to bump the `libp2p-identity` version in `libp2p-core` or any of the other crates below? Given that we will release a patch of `libp2p-identity`, downstream consumers can simply upgrade their `Cargo.lock`. It doesn't require any intermediate crates to upgrade. Am I missing something?

Reply:
I understood there was no downside to upgrading them to use the latest version but I can change them back